src/script/user/AppLockRepository.ts

```diff
@@ -99,7 +96,12 @@ export class AppLockRepository {
     }
   };
 
-  setCode = async (code: string): Promise<void> => {
+  public setDisabled = () => {
```

Comment:
There was no way to disable the applock without showing additional modals, now there is